src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.impl.DataConverter;
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.reader.Reader;
+import core.basesyntax.reader.ReaderImpl;
+import core.basesyntax.report.ReportGenerator;
+import core.basesyntax.report.impl.ReportGeneratorImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.BalanceOperation;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import core.basesyntax.service.impl.PurchaseOperation;
+import core.basesyntax.service.impl.ReturnOperation;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.service.impl.SupplyOperation;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.writer.Writer;
+import core.basesyntax.writer.impl.WriterImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        Reader fileReader = new ReaderImpl();
+        List<List<String>> inputReport = fileReader.read("src/main/resources/reportToRead.cvs");
```

Comment:
The file extension in the `read` method call is incorrect. It should be `.csv` instead of `.cvs`. Please change `"src/main/resources/reportToRead.cvs"` to `"src/main/resources/reportToRead.csv"` to match the standard CSV format.